### ISSUE TITLE
Accessible Chat

### ISSUE THREAD

**Issue by Terreii** [Terreii/andromeda-viewer#277]
**Is your feature request related to a problem? Please describe.**
The chat correctly not accessible. If it is in focus, all of it is read out! Blind users can't only select once message.

**Describe the solution you'd like**
Change the chat into a [ARIA-Grid](https://w3c.github.io/aria-practices/#grid "W3C's aria Practices Grid section").

**Describe alternatives you've considered**
Change all messages to be focus-able. But this would be a bad solution.

**Additional context**
A working example would be the ["Example 3: Scrollable Search Results"](https://w3c.github.io/aria-practices/examples/grid/LayoutGrids.html#ex3_label).

**Comment by Terreii** [comment]
More info from MDN: [Grid role](https://developer.mozilla.org/de/docs/Web/Accessibility/ARIA/Roles/Grid_Role).

**Comment by Terreii** [comment]
Or the [log role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_log_role) or the [feed role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Feed_Role).
